infrastructure/private-api-config.yaml

```diff
@@ -0,0 +1,80 @@
+plugin: openapi
+specFile: ./private-api.yaml
+
+validation:
+  request: log
+
+resources:
+  - method: GET
+    path: /authorization
+    requestHeaders:
+      SessionHeader: session-id
```

Comment:
`SessionHeader` is just the term we're using to reference in that spec - it's not something that actually goes into the query. To achieve this we need to do:
```
requestHeaders:
    session-id: <whatever you want to match on>
```